backend/substrapp/tasks/tasks.py

```diff
@@ -939,16 +944,15 @@ def save_models(subtuple_directory, tuple_type, subtuple_key):
         }
 
     elif tuple_type == COMPOSITE_TRAINTUPLE_TYPE:
-
-        for type_model, filename in [('end_head_model', OUTPUT_HEAD_MODEL_FILENAME),
-                                     ('end_trunk_model', OUTPUT_TRUNK_MODEL_FILENAME)]:
+        for t_type, filename, hash_salt in [('end_head_model', OUTPUT_HEAD_MODEL_FILENAME, HASH_KEY_SUFFIX_HEAD),
```

Comment:
could you explain what does `t_type` stands for and why `type_model` was renamed?

Reply:
`type_model` has been renamed because of flake8 (line length).

The new name `t_type` means "tuple type" but that's not a good name. Let me rename it to `m_type` (model type) which is more accurate.

Reply:
done https://github.com/SubstraFoundation/substra-backend/pull/298/commits/c11b42b54388441d941d144e542708e893e37bda